app/controllers/infra_networking_controller.rb

```diff
@@ -110,6 +133,15 @@ def explorer
     render :layout => "application"
   end
 
+  def index
+    @lastaction = "index"
+
+    build_accordions_and_trees
```

Comment:
I realize this PR is still WIP, but deploraization should effectively remove accordions and trees entirely.  Accordions would instead become separate menu items and each menu item would require separate controllers/views.  Trees have to be discussed...in many places tree structures are _really_ needed and can just be normal list-view tables, however in some places we still may need trees (as in the case of infra-networking), so we need to figure out how to present that.

cc @jeffibm